Commander.Cli/Program.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Cqse.Teamscale.Profiler.Commons.Ipc;
+using System.Reflection;
+
+var profilerIpc = new ProfilerIpc(new IpcConfig());
+
+while(true)
+{
+    Command nextCommand = Command.Start;
+    if (!string.IsNullOrEmpty(profilerIpc.TestName))
+    {
+        nextCommand = Command.Stop;
+    }
+
+    (Command command, string? arg) = WaitForCommand(nextCommand);
+    if (ExecuteCommand(command, arg))
+    {
+        return;
+    }
+}
+
+bool ExecuteCommand(Command command, string? arg)
+{
+    switch(command)
+    {
+        case Command.Start:
+            profilerIpc.StartTest(arg);
+            Help($"Started test {arg}");
+            return false;
+        case Command.Stop:
+            if (arg?.Length > 1)
+            {
+                arg = char.ToUpper(arg[0]) + arg.Substring(1).ToLower();
+                if (Enum.TryParse(arg, out TestExecutionResult result))
+                {
+                    profilerIpc.EndTest(result);
+                    Help($"Stopped test with result {arg}");
+                }
+
+                return false;
+            }
+            Help($"Unknown test result {arg}, use one of these values: {CommandAttribute.ValidResultValues}");
+            return false;
+        case Command.Exit:
+            return true;
+        default:
+            return false;
+    }
+}
+
+(Command, string?) WaitForCommand(params Command[] commands)
```

Comment:
It can be called using a single argument (due to `params`). The method implementation makes use multiple commands anyways (due to addition of exit - `commands.Append(Command.Exit)`), so there is not that much to gain. and if we'd add e.g. a cancel (incl throw away of data) we're set.